probe/ssh/endpoint.go

```diff
@@ -82,8 +82,8 @@ func (e *Endpoint) SSHConfig(kind, name string, timeout time.Duration) (*ssh.Cli
 		// Create the Signer for this private key.
 		var signer ssh.Signer
 
-		if len(e.PrivatePassword) > 0 {
-			signer, err = ssh.ParsePrivateKeyWithPassphrase(key, []byte(e.PrivatePassword))
+		if len(e.Passphrases) > 0 {
+			signer, err = ssh.ParsePrivateKeyWithPassphrase(key, []byte(e.Passphrases))
```

Comment:
```suggestion
			signer, err = ssh.ParsePrivateKeyWithPassphrase(key, []byte(e.Passphrase))
```

Reply:
sorry， my english is not good  :(

Reply:
no need to say sorry, I guide you to the improper word. ;-)

Reply:
it's better than `private_password` :-)